ledger/sandbox-classic/src/main/scala/platform/sandbox/Cli.scala

```diff
@@ -33,6 +33,12 @@ private[sandbox] object Cli extends SandboxCli {
           s" Also note that instructing $Name to load a scenario will have the side effect of loading _all_ the .dar files provided eagerly (see --eager-package-loading)."
       )
     parser
+      .opt[String]("sql-backend-jdbcurl")
+      .optional()
+      .text(
+        s"Deprecated: Use the DAML Driver for PostgreSQL if you need persistence.\nThe JDBC connection URL to a Postgres database containing the username and password as well. If present, $Name will use the database to persist its data.")
```

Comment:
is it intentionally `$Name`, not `$name` here?

Reply:
Yes, `Name` is the argument we pass to the `CommonCli` thingy in which it is then bound to `name`.